[libc] libc.test.src.setjmp.setjmp_test is failing

LOG
```
[1/1] Running unit test libc.test.src.setjmp.setjmp_test
FAILED: projects/libc/test/src/setjmp/CMakeFiles/libc.test.src.setjmp.setjmp_test /home/schrodingerzy/Documents/llvm/llvm-project/build/projects/libc/test/src/setjmp/CMakeFiles/libc.test.src.setjmp.setjmp_test 
cd /home/schrodingerzy/Documents/llvm/llvm-project/build/projects/libc/test/src/setjmp && /home/schrodingerzy/Documents/llvm/llvm-project/build/projects/libc/test/src/setjmp/libc.test.src.setjmp.setjmp_test.__build__
[ RUN      ] LlvmLibcSetJmpTest.SetAndJumpBack
ninja: build stopped: subcommand failed.
```
LLDB
```
❯ lldb /home/schrodingerzy/Documents/llvm/llvm-project/build/projects/libc/test/src/setjmp/libc.test.src.setjmp.setjmp_test.__build__
(lldb) target create "/home/schrodingerzy/Documents/llvm/llvm-project/build/projects/libc/test/src/setjmp/libc.test.src.setjmp.setjmp_test.__build__"
Current executable set to '/home/schrodingerzy/Documents/llvm/llvm-project/build/projects/libc/test/src/setjmp/libc.test.src.setjmp.setjmp_test.__build__' (x86_64).
(lldb) r
Process 1069996 launched: '/home/schrodingerzy/Documents/llvm/llvm-project/build/projects/libc/test/src/setjmp/libc.test.src.setjmp.setjmp_test.__build__' (x86_64)
[ RUN      ] LlvmLibcSetJmpTest.SetAndJumpBack
Process 1069996 stopped
* thread #1, name = 'libc.test.src.s', stop reason = signal SIGSEGV: invalid address (fault address: 0x0)
    frame #0: 0x00005555555552d5 libc.test.src.setjmp.setjmp_test.__build__`LlvmLibcSetJmpTest_SetAndJumpBack::Run() [inlined] bool __llvm_libc_19_0_0_git::testing::Test::test<int, 0>(this=0xaaaaaaaaaaaaaaaa, Cond=EQ, LHS=124, RHS=124, LHSStr="longjmp_called", RHSStr="n", Loc=(file = "/home/schrodingerzy/Documents/llvm/llvm-project/libc/test/src/setjmp/setjmp_test.cpp", line = 33)) at LibcTest.h:135:27
   132                         int> = 0>
   133    bool test(TestCond Cond, ValType LHS, ValType RHS, const char *LHSStr,
   134              const char *RHSStr, internal::Location Loc) {
-> 135      return internal::test(Ctx, Cond, LHS, RHS, LHSStr, RHSStr, Loc);
   136    }
   137 
   138    template <typename ValType,
(lldb) bt
* thread #1, name = 'libc.test.src.s', stop reason = signal SIGSEGV: invalid address (fault address: 0x0)
  * frame #0: 0x00005555555552d5 libc.test.src.setjmp.setjmp_test.__build__`LlvmLibcSetJmpTest_SetAndJumpBack::Run() [inlined] bool __llvm_libc_19_0_0_git::testing::Test::test<int, 0>(this=0xaaaaaaaaaaaaaaaa, Cond=EQ, LHS=124, RHS=124, LHSStr="longjmp_called", RHSStr="n", Loc=(file = "/home/schrodingerzy/Documents/llvm/llvm-project/libc/test/src/setjmp/setjmp_test.cpp", line = 33)) at LibcTest.h:135:27
    frame #1: 0x00005555555552b2 libc.test.src.setjmp.setjmp_test.__build__`LlvmLibcSetJmpTest_SetAndJumpBack::Run(this=0xaaaaaaaaaaaaaaaa) at setjmp_test.cpp:33:3
    frame #2: 0x000055555555ddc1 libc.test.src.setjmp.setjmp_test.__build__`__llvm_libc_19_0_0_git::testing::Test::runTests(TestFilter=0x0000000000000000) at LibcTest.cpp:147:8
    frame #3: 0x00007ffff79f8010 libc.so.6`__libc_start_call_main(main=(libc.test.src.setjmp.setjmp_test.__build__`main at LibcTestMain.cpp:16:33), argc=1, argv=0x00007fffffffe438) at libc_start_call_main.h:58:16
    frame #4: 0x00007ffff79f80ca libc.so.6`__libc_start_main_impl(main=(libc.test.src.setjmp.setjmp_test.__build__`main at LibcTestMain.cpp:16:33), argc=1, argv=0x00007fffffffe438, init=<unavailable>, fini=<unavailable>, rtld_fini=<unavailable>, stack_end=0x00007fffffffe428) at libc-start.c:360:3
    frame #5: 0x0000555555555165 libc.test.src.setjmp.setjmp_test.__build__`_start + 37
(lldb) frame info
frame #0: 0x00005555555552d5 libc.test.src.setjmp.setjmp_test.__build__`LlvmLibcSetJmpTest_SetAndJumpBack::Run() [inlined] bool __llvm_libc_19_0_0_git::testing::Test::test<int, 0>(this=0xaaaaaaaaaaaaaaaa, Cond=EQ, LHS=124, RHS=124, LHSStr="longjmp_called", RHSStr="n", Loc=(file = "/home/schrodingerzy/Documents/llvm/llvm-project/libc/test/src/setjmp/setjmp_test.cpp", line = 33)) at LibcTest.h:135:27
(lldb) frame variable
(__llvm_libc_19_0_0_git::testing::Test *) this = 0xaaaaaaaaaaaaaaaa
(__llvm_libc_19_0_0_git::testing::TestCond) Cond = EQ
(int) LHS = 124
(int) RHS = 124
(const char *) LHSStr = 0x0000555555565040 "longjmp_called"
(const char *) RHSStr = 0x0000555555565145 "n"
(__llvm_libc_19_0_0_git::testing::internal::Location) Loc = (file = "/home/schrodingerzy/Documents/llvm/llvm-project/libc/test/src/setjmp/setjmp_test.cpp", line = 33)
(lldb) 
```

--- a/libc/src/setjmp/x86_64/CMakeLists.txt
+++ b/libc/src/setjmp/x86_64/CMakeLists.txt
@@ -9,6 +9,11 @@ add_entrypoint_object(
   COMPILE_OPTIONS
     -O3
     -fno-omit-frame-pointer
+    # TODO: Remove once one of these lands:
+    # https://github.com/llvm/llvm-project/pull/87837
+    # https://github.com/llvm/llvm-project/pull/88054
+    # https://github.com/llvm/llvm-project/pull/88157
+    -ftrivial-auto-var-init=uninitialized
 )
 
 add_entrypoint_object(

